### DISCUSSION TITLE
How can I get the PRO tag in Github profile?

### DISCUSSION THREAD

**Discussion by Huseyin-Cinar** [community/community#19414] [Other Feature Feedback, Questions, & Ideas] [answered]
Can I earn the "PRO" tag we see on Github profiles? How do I win if I can? I will be glad if you help me :)

**Comment by melvincwng** [comment, marked as answer]
Hi @Huseyin-Cinar, to get the Github Pro badge, please follow the steps in this [Github Docs webpage](https://docs.github.com/en/billing/managing-billing-for-your-github-account/upgrading-your-github-subscription).

It should cost $4 per month for the Github Pro subscription. Verified students can get it for free when they sign up for the Github Student Developer Pack.

Hope this clarifies.

Thank you and have a nice day 😊

**Reply by Huseyin-Cinar** [reply]
I tried to join the Github Sudent Developer Pack but failed :(

Can you give some information about how it's done?

**Reply by melvincwng** [reply]
Hello @Huseyin-Cinar, to apply for the Github Student Developer Pack, kindly follow the steps in this [Github Docs webpage](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack).

That webpage contains detailed step by step instructions on how to apply for the Github Student Pack so that you can unlock the 'Pro' badge for free.

Hope this helps and clarifies.

**Reply by Huseyin-Cinar** [reply]
Thank you my friend :)

have a nice day

**Reply by melvincwng** [reply]
@Huseyin-Cinar, pleasure to help 😄

**Reply by Manuel2u** [reply]
i have a pro badge but it is not showing on my highlights why??

**Comment by ertugrul37** [comment]
https://github.com/pricing buradan yapılıyor

**Reply by Huseyin-Cinar** [reply]
Teşekkür ederim ertuğrul

**Comment by myConsciousness** [comment]
You just need to contract with PRO plan :)

But note that this PRO tag will be gone if you cancel it.

**Comment by dikkyryan** [comment]
Verified students can get it for free
